File: src/entity/bullet/RegularBullet.java
package entity.bullet;

import entity.enemy.Enemy;
import javafx.scene.image.Image;

public class RegularBullet extends Bullet {
	
	public static final int DAMAGE = 1;
	public static final int SPEED = 4;
	public static final String IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile272.png";
	
	public RegularBullet(double x, double y, Enemy enemy) {
		super(x, y, enemy, SPEED, DAMAGE);
		
		img = new Image(IMAGE_URL);
	}

}

File: src/entity/bullet/RegularBullet.kt
package entity.bullet

import entity.enemy.Enemy
import javafx.scene.image.Image

class RegularBullet(x: Double, y: Double, enemy: Enemy) : Bullet(x, y, enemy, RegularBullet.SPEED, RegularBullet.DAMAGE) {
    override var img: Image = Image(IMAGE_URL)

    companion object {
        val DAMAGE = 1
        val SPEED = 4.0
        val IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile272.png"
    }

}
